src/openfl/text/TextField.hx

```diff
@@ -3187,6 +3191,54 @@ class TextField extends InteractiveObject
 			scrollV -= event.delta;
 		}
 	}
+	
+	@:noCompletion private function this_onDoubleClick(event:MouseEvent):Void
+	{
+		if (__textEngine.selectable) {
+			__updateLayout();
+			
+			var delimiters:Array<String> = ['\n', '.', '!', '?', ',', ' ', ';', ':', '(', ')', '-', '_', '/'];
+			
+			var txtStr:String = __text;
+			var leftPos:Int = -1;
+			var rightPos:Int = txtStr.length;
+			var pos:Int = 0;
+			var startPos:Int = Std.int( Math.max(__caretIndex, 1) );
+			if (txtStr.length > 0 && __caretIndex >= 0 && rightPos >= __caretIndex)
+			{
+				for(c in delimiters)
+				{
+					pos = txtStr.lastIndexOf(c, startPos - 1);
+					if (pos > leftPos) leftPos = pos + 1;
+
+					pos = txtStr.indexOf(c, startPos);
+					if (pos < rightPos && pos != -1) rightPos = pos;
+				}
+				
+				if (leftPos != rightPos)
+				{
+					__selectionIndex = leftPos;
+					__caretIndex = rightPos;
+
+					#if openfl_html5
+					if (DisplayObject.__supportDOM)
+					{
+						if (__renderedOnCanvasWhileOnDOM)
+						{
+							__forceCachedBitmapUpdate = true;
+						}
+					}
+					else
+					#end
+					{
```

Comment:
The { } don't line up with anything in the case of !js

Reply:
Oops. Fixed it. While I was in there, I noticed the same error in stage_onMouseMove, btw. Unless you see a reason to leave it be, I'll fix that one too.

Reply:
I'll just do that one to keep the PR clean if it ever needs to be reverted